website/docs/r/dms_endpoint.html.markdown

```diff
@@ -43,7 +43,7 @@ resource "aws_dms_endpoint" "test" {
 The following arguments are supported:
 
 * `certificate_arn` - (Optional, Default: empty string) The Amazon Resource Name (ARN) for the certificate.
-* `database_name` - (Optional) The name of the endpoint database.
+* `database_name` - (Required) The name of the endpoint database.
```

Comment:
Hi @gifkoek-fargo 👋 This argument is optional for many of the `engine_name` values. Please leave it optional, but feel free to insert additional information in the description.

```suggestion
* `database_name` - (Optional) The name of the endpoint database. Certain engines require this configuration, such as `aurora` and `aurora-postgresql`.
```

We hope to introduce better plan-time validation for this class of problem in the future.